src/store/api/leaderboard/index.js

```diff
@@ -0,0 +1,34 @@
+import { createApi } from "@reduxjs/toolkit/query/react";
+import baseQuery from "../base";
+import transformLeaderboardData from "./transformer";
+
+export const leaderboardApi = createApi({
+  reducerPath: "leaderboardApi",
+  baseQuery,
+  endpoints: (builder) => ({
+    fetchLeaderboard: builder.query({
+      query: ({ round, ghostLegion }) => `/api/leaderboard?round=${round}&ghost_legion=${ghostLegion}`,
+      transformResponse: transformLeaderboardData
+    }),
+    fetchPastLeaderboards: builder.query({
+      queryFn: async ({ year = 2023, round, ghostLegion, page, filters, sorts }) => {
+        let path = round == 1 ? "round1" : ghostLegion == true ? "ghost-legion" : "final";
```

Comment:
Let's make this triple equals else the linter will fail

Reply:
Done